app/scripts/modules/core/src/serverGroup/ServerGroupHeader.tsx

```diff
@@ -72,7 +72,16 @@ export class ImageList extends React.Component<IServerGroupHeaderProps, IImageLi
   }
 
   public render() {
-    const images = this.props.images.sort();
+    const shorten = function(val: string): string {
+      const sha = val.lastIndexOf('sha256:');
+      if (sha > 0) {
+        // 14 = "sha256:" + 7 characters of hash
+        return val.substr(0, sha + 14) + '...';
```

Comment:
I think `String.prototype.substr` is [considered legacy](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr) so let's use `String.prototype.substring` if possible.